src/enclave/rpc_sessions.h

```diff
@@ -191,6 +205,9 @@ namespace enclave
       // this type of session to be rare and want it to succeed even when we are
       // busy.
       sessions.insert(std::make_pair(id, session));
+
+      sessions_peak = std::max(sessions_peak, sessions.size());
```

Comment:
Should we separate client from server sessions in the stats?

Reply:
Unless we give them different quotas, it won't make a much difference to an operator, particularly since they have little ability to influence client sessions.

Reply:
Since client session are not affected by the session limits, I think we shouldn't count them here? Otherwise, we could technically report a number of open session that's actually over the limit.

Reply:
So our caps count both kinds of session because both affect our fds, and could eventually cause the node to crash. If we ever create several hundred client sessions at the same time. Although the operator doesn't currently have any dials to turn to _stop_ the node creating client sessions, I think they want to see the same warning signal in this case ("your active sessions are at or near the cap, so users are going to get errors"). Separating these counts is an extra complication, and (I think) hides the information the operator cares about in the common case.

In practice, there's only ever 1 client session at a time currently (when and if we add more, we can work out how _they're_ capped and maybe how they're reported here), and I think its fine to (correctly) report that the open sessions are over the given limit if this happens. Its an artifact of the fact we don't apply the limit to client sessions, and there's no motivation to revisit that here.

Reply:
@eddyashton That makes sense to me, and as you said, even with the JWT refresh job, the number of clients is only ever going to be one, at least as of 1.x.